src/RuleSet.php

```diff
@@ -38,7 +38,7 @@
             'line_ending' => true,
             'lowercase_constants' => true,
             'lowercase_keywords' => true,
-            'method_argument_space' => true,
+            'method_argument_space' => ['ensure_fully_multiline' => true],
```

Comment:
put `'method_argument_space' => true,` to `@Symfony` and `'method_argument_space' => ['ensure_fully_multiline' => true],` to `.php_cs.dist`